jekyll/_cci2/configuration-reference.md

```diff
@@ -484,33 +498,72 @@ jobs:
 Class             | vCPUs | RAM
 ------------------|-------|-----
 medium (default)  | 4     | 15GB
+large             | 8     | 30GB
+xlarge            | 16    | 60GB
 {: class="table table-striped"}
 
-There is currently only one size of Windows Machine available, please let us know if you find yourself needing more.
-
 ###### Example Usage
 ```yaml
 version: 2.1
 
 orbs:
-  win: circleci/windows@1.0.0
+  win: circleci/windows@2.3.0
 
 jobs:
   build:
-    executor: win/vs2019
+    executor: win/default
     steps:
       ... // other config
 ```
 
+See the [Windows Getting Started document]({{ site.baseurl }}/2.0/hello-world-windows/) for more details and examples of using the Windows executor.
+
 ##### GPU Executor (Linux)
 
-Class            | vCPUs | Memory (GiB) | GPUs | GPU Memory (*GiB)
------------------|-------|--------------|------|-----------------
-1GPU\*           | 16    | 122GiB       | 1    | 8
-2GPU\*           | 32    | 244GiB       | 2    | 16
-4GPU\*           | 64    | 488GiB       | 4    | 32
+Class               | vCPUs | RAM | GPUs |    GPU model    | GPU Memory (GiB)
+--------------------|-------|-----|------|-----------------|------------------
+gpu.nvidia.small\*  |   4   | 15  | 1    | Nvidia Tesla P4 | 8
+gpu.nvidia.medium\* |   8   | 30  | 1    | Nvidia Tesla T4 | 16
 {: class="table table-striped"}
 
+###### Example Usage
+```yaml
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      resource_class: gpu.nvidia.small
+      image: ubuntu-1604-cuda-10.1:201909-23
+    steps:
+      ... // other config
+```
+
+See the [Available Linux GPU images](#available-linux-gpu-images) section for the full list of available images.
+
+##### GPU Executor (Windows)
+
+Class                       | vCPUs | RAM | GPUs |    GPU model    | GPU Memory (GiB)
+----------------------------|-------|-----|------|-----------------|------------------
+windows.gpu.nvidia.medium\* |   8   | 30  | 1    | Nvidia Tesla T4 | 16
```

Comment:
I'd recommend using the `<sup>` with a number rather than the `\*`  tag as used in [other tables](http://localhost:4000/docs/2.0/configuration-reference/#executors-requires-version-21) across this doc